manager/assets/modext/widgets/windows.js

```diff
@@ -258,6 +258,10 @@ MODx.window.CreateCategory = function(config) {
             ,id: 'modx-'+this.ident+'-rank'
             ,xtype: 'numberfield'
             ,anchor: '100%'
+        },{
```

Comment:
Thanks, I'll test this option.

Reply:
Yes, it looks interesting.
Old version with `xtype: 'label'`:

![category_label](https://user-images.githubusercontent.com/12523676/153387945-acacdd43-1efe-4def-a58f-2b66ed5db813.png)

New version with `xtype: 'modx-description'`:

![category_desc](https://user-images.githubusercontent.com/12523676/153387940-e7157668-05af-4ec0-9085-ff7046d54484.png)

Reply:
I think you're good to go with this.

Reply:
Sorry to chime in late here. Not that we have an official style guide written anywhere (besides the actual Lexicon files themselves) but Element(s) is a proper noun in MODX and should be capitalized when referenced as we do with all other MODX-specific terms such as Snippet, Chunk, Template, etc.

Avoiding the passive voice a variation and an edit: 

```
Categories will appear in the Categories folder once one or more Elements are assigned to that Category.
```

```
Note that, until they are assigned to one or more Elements, you will only see the Categories you create listed in the Categories folder.
```

I’m not sure the “Note that” is required here. I lean on being as direct as possible. But, I’m a 3/10 on that.

Reply:
@jaygilmore - The first variation isn't quite right, it should be:
`"A Category will appear in an Element’s folder once one or more of its Elements are assigned to that Category."`
The lean toward adding the word “only” (e.g., “will only appear”) comes from the desire to be really clear about the required condition, even though it's technically not needed.

Reply:
@smg6511 @jaygilmore I want to choose this variant:

`A Category will appear in an Element’s folder once one or more of its Elements are assigned to it (by default, all Categories, even empty ones, are displayed in the Categories section).`

Is this okay with everyone?

p.s. @jaygilmore You do not indicate that all categories, even empty ones, are in the Categories section, and this is important, because the user can create a bunch of unnecessary categories without knowing it. I added this.

Reply:
That sounds good to me!

Reply:
@Ruslan-Aleev @smg6511 good points there. Remind me what the `Element’s folder` refers to in the UI?

Reply:
Chunks, Templates, TVs, Plugins, and Snippet folders

Reply:
@jaygilmore I didn’t quite understand the question, here the `Element’s folder` means “Any sections of elements” (templates, chunks, TV, etc.) Or are you asking about something else?

To be honest, I like the variant earlier more, it is clearer, in my opinion.

`A Category will only be displayed if at least one Element is assigned to it (by default, all Categories are displayed in the Categories section).`